### ISSUE TITLE
PropertimeFitter is deprecated

### ISSUE THREAD

**Issue by alexpearce** [alexpearce/CharmProduction#59]
When creating ntuples, I noticed this in the logs:

```
ToolSvc.properTimeWARNING PropertimeFitter is no longer maintained and thus depreciated.
```

This is because we've got lines like this:

```python
mother_loki_vars = {
    # ...
    'Loki_BPVLTIME': "BPVLTIME('PropertimeFitter/properTime:PUBLIC')",
}
```

Instead we should just be using `BPVLTIME()`. This isn't a big problem for exsiting ntuples, but for new ones we should make sure to switch. (The stripping lines, see #53, do use `BPVLTIME()`, but for the MVA training we use this deprecated fitter, and so also use it in the MVA cut.)

**Comment by chrisburr** [comment]
We can fix this when we get stripping 21, we also need to add `DOCAMIN` as Vava is using it in the HLT selection.

**Comment by alexpearce** [comment]
Ah yes I forgot we'll reprocess for S21. Good plan.

**Comment by chrisburr** [comment]
How can we get `DOCAMIN`? I can [see](https://twiki.cern.ch/twiki/bin/view/LHCb/LoKiParticleFunctions#MINCLAPP_C_type_LoKi_Particles_M) some [possibilities](https://twiki.cern.ch/twiki/bin/view/LHCb/LoKiParticleFunctions#AMAXDOCA_AMINDOCA_Distance_of_cl) but I can't get them to work.

**Comment by alexpearce** [comment]
Have tried just using the functor `DOCAMIN`?

**Comment by chrisburr** [comment]
Yeah [it doesn't exist.](http://lhcb-release-area.web.cern.ch/LHCb-release-area/DOC/davinci/releases/latest/doxygen/d7/dae/namespace_lo_ki_1_1_cuts.html)

**Comment by alexpearce** [comment]
[This looks helpful](https://groups.cern.ch/group/lhcb-davinci/Lists/Archive/Flat.aspx?RootFolder=%2fgroup%2flhcb-davinci%2fLists%2fArchive%2fTupling%20AMAXDOCA&FolderCTID=0x01200200B795ECDBD8943F4E90E5F02A017196C9). During testing, I'd recommend monitoring both `LoKi.Particles.PFunA(AMAXDOCA(''))` and `LoKi.Particles.PFunA(AMINDOCA(''))`, comparing our usual `DOCAMAX` to the new one, so we know this `PFunA` thing is doing stuff correctly.

**Comment by chrisburr** [comment]
Ah perfect, I didn't think to search the mailing list.

**Comment by alexpearce** [comment]
Yep looks good to me.

Andrea Contu pointed out that our BDT's are using `PropertimeFitter` as the fitter for the proper time input variable. I have [removed](https://github.com/alexpearce/CharmProduction/commit/93d3fe01a731c3a22ec4a1b2b4bbf4be2af690ff) those references in [`stripping-lines`](https://github.com/alexpearce/CharmProduction/tree/stripping-lines).
